config/routes.rb

```diff
@@ -162,11 +162,14 @@
     match 'reindex/:pid',   :action => :reindex, :as => 'reindex', :via => [:get, :post]
     match 'delete_from_index/:pid', :action => :delete_from_index, :via => [:get, :post]
     get 'index_exceptions'
-    resources :objects
+    namespace :objects do
```

Comment:
If we are implementing create only, maybe:
`resources :objects, only: :create`
?

That way it keeps the resourceful structure without just tacking it on a namespace?